test/validation/api/random/random.c

```diff
@@ -230,9 +230,20 @@ static void random_test_frequency(odp_random_kind_t kind)
 		8929.74, 17419.3, 34224.0, 67587.1,
 	};
 
+	int bits_max = 8;
+
+#ifdef __ARM_FEATURE_RNG
```

Comment:
I'm not that fond of adding architecture specific flags into the validation tests. Perhaps the same result could be achieved by measuring time and capping the test length to some maximum value?

Reply:
What I don't like about that is that you end up doing time measurements even when they're not needed, which is in most environments. But in practice it doesn't matter. Done in v2.